fix(overlay): fixed overlay issue in 'card-overlay' component

raiyanu opened the pull request:
![Text particles error](https://github.com/user-attachments/assets/3f8dd981-b85f-4c7a-b87d-2b4979176cf1)

raiyanu committed:
fix(overlay): fixed overlay issue in 'card-overlay' component

raiyanu committed:
Update demo.ts

swapnilsparsh reviewed (changes requested):
### It is not centre aligned and resolve the conflicts


![image](https://github.com/user-attachments/assets/c73d092a-3086-41ed-bc30-06fa770ad186)

raiyanu committed:
Merge branch 'master' into master

raiyanu committed:
chore(layout): Remove max-width constraint to center <BODY/>

raiyanu commented:
@swapnilsparsh  I've addressed the feedback:
- Fixed the center alignment issue
- Resolved the merge conflicts

The changes are ready for another review. Let me know if you need anything else!

swapnilsparsh reviewed (changes requested):
### Not responsive

![image](https://github.com/user-attachments/assets/0d12aea5-68d8-4dbd-b6f1-a480c87b5666)
